### PR TITLE
Separated Lint tests

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@ python:
   - "2.7"
 
 env:
- - VERSION="8.0" ODOO_REPO="odoo/odoo"
- - VERSION="8.0" ODOO_REPO="OCA/OCB"
+  - VERSION="8.0" LINT_CHECK="1"
```
Comment (pedrobaeza) [inline .travis.yml:7]: Extra space

**Comment by yvaucher** [comment]
@pedrobaeza the space is intended. I wanted to have 2 spaces everywhere. It will ease automation of `travis.yml` update

**Comment by yvaucher** [comment]
re-pushed to trigger travis

**Comment by pedrobaeza** [comment]
OK, thanks for the clarification :+1:

**Comment by guewen** [comment]
:+1:

**Pull request merged**